pkg/dhclient/dhcp4.go

```diff
@@ -172,8 +175,8 @@ func (p *Packet4) Boot() (*url.URL, error) {
 	}
 
 	if len(u.Scheme) == 0 {
-		// Defaults to tftp is not specified.
-		u.Scheme = "tftp"
+		// Use the DefaultScheme if not specified
+		u.Scheme = DefaultScheme
```

Comment:
I'm wondering why this differs between dhcpv4 and dhcpv6?

Reply:
Not sure why but it seems all dhcpv6 I've encountered pass the full url with scheme. It seems it just wasn't needed